ani-cli

```diff
@@ -511,27 +481,24 @@ while getopts 'viq:dp:chDUVa:' OPT; do
 			exit 0
 			;;
 		d)
-			is_download=1
+			is_download='1'
```

Comment:
don't know numbers, we will later use these stuff in `-eq` statements in test. Its better you leave it as is, makes it evident that the variable is a number

Reply:
You are absolutely right, I'll revert them